app/views/foreman/job_templates/install_packages_by_search_query_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+kind: job_template
+name: Install packages by search query - Katello Ansible Default
+job_category: Katello
+description_format: 'Install package(s) %{Package search query}'
+feature: katello_package_install_by_search
+provider_type: Ansible
+template_inputs:
+- name: Package search query
+  description: Filter criteria for packages to be installed. IMPORTANT- If left blank, the job will attempt to install all possible packages.
+  input_type: user
+  required: false
+%>
+<% package_names = @host.package_names_for_job_template(
+  action: 'install',
+  search: input('Package search query')
+) -%>
+---
+- hosts: all
+  tasks:
+    - package:
+        name: 
+<% package_names.each do |package_name| -%>
+          - <%= package_name %>
+<% end -%>
+        state: present
```

Comment:
```suggestion
<%= render_template('Package Action - Ansible Default', :state => 'present', :name => package_names) %>
```

At least it will be on par with the [Katello without Ansible](https://github.com/Katello/katello/blob/1e385f23bbc9d95c112f8a91d99f04d2bae7389c/app/views/foreman/job_templates/install_packages_by_search_query.erb)